zebrad/src/components/sync.rs

```diff
@@ -421,21 +410,21 @@ where
                 }
 
                 // Once we're below the lookahead limit, we can keep extending the tips.
-                tracing::info!(
+                debug!(
```

Comment:
This log is really helpful for diagnosing sync issues. I'd like to keep it at info level.